operators/src/upgrade/common/error.rs

```diff
@@ -1,18 +1,78 @@
+use std::string::FromUtf8Error;
+
+use serde::Serialize;
 /// Helm error to be used for client.
-#[derive(Debug, Clone)]
-pub(crate) enum Error {
+use snafu::Snafu;
+
+/// Contains Errors that may generate while execution of k8s_client.
+#[derive(Debug, Snafu)]
+#[snafu(visibility(pub), context(suffix(false)))]
+#[allow(clippy::enum_variant_names)]
+pub enum Error {
     /// Helm not installed error.
-    HelmNotInstalled(String),
+    #[snafu(display("Helm not installed, error: {}", source))]
+    HelmNotInstalled { source: anyhow::Error },
+
     /// Error while running helm command.
-    HelmStd(String),
+    #[snafu(display("{}", source))]
+    HelmStd { source: anyhow::Error },
```

Comment:
These errors here seem to be just strings, so we can simply add the string to the display.